test/unit/CustomFunction.spec.ts

```diff
@@ -181,119 +98,16 @@ describe('CustomFunction class', () => {
       it('fail should call functions.completeError', async () => {
         const client = new WebClient('sometoken');
         const completeMock = sinon.stub(client.functions, 'completeError').resolves();
-        const complete = CustomFunction.createFunctionFail(
-          { isEnterpriseInstall: false, functionExecutionId: 'Fx1234' },
-          client,
-        );
+        const complete = createFunctionFail({ isEnterpriseInstall: false, functionExecutionId: 'Fx1234' }, client);
         await complete({ error: 'boom' });
         assert(completeMock.called, 'client.functions.completeError not called!');
```

Comment:
```suggestion
        assert(completeMock.called, 'client.functions.completeError not called!');
        assert(
          completeMock.calledWith({ function_execution_id: 'Fx1234', error: 'boom' }),
          'client.functions.completeError called with unexpected arguments!',
        );
```

Similar recommendation for an added assert here, and it'd also be super nice to change `complete` to `fail` but that's no blocker!